efel/cppcore/LibV2.cpp

```diff
@@ -601,173 +518,17 @@ static int __fast_AHP_change(const vector<double>& fastahp,
 int LibV2::fast_AHP_change(mapStr2intVec& IntFeatureData,
                            mapStr2doubleVec& DoubleFeatureData,
                            mapStr2Str& StringData) {
-  int retval;
-  vector<double> fastahp;
-  retval = getVec(DoubleFeatureData, StringData, "fast_AHP", fastahp);
-  if (retval < 0) return -1;
+  const auto& features = getFeatures(DoubleFeatureData, {"fast_AHP"});
+  const vector<double>& fastahp = features.at("fast_AHP");
   vector<double> fastahpchange;
-  retval = __fast_AHP_change(fastahp, fastahpchange);
-  if (retval >= 0) {
-    setVec(DoubleFeatureData, StringData, "fast_AHP_change",
-                 fastahpchange);
+  int retval = __fast_AHP_change(fastahp, fastahpchange);
+  if (retval > 0) {
+    setVec(DoubleFeatureData, StringData, "fast_AHP_change", fastahpchange);
   }
   return retval;
 }
 // end of fast_AHP_change
 
-// *** E6 ***
```

Comment:
Feature starting with E such as E6, here seem to be duplicates of existing features with wildcard for APWaveform.
We should mention in PR why each feature was removed by mentioning the duplicates and other features that could be used for the duplicate features.

Reply:
Thanks, I am putting this sentence to the description.